response_operations_ui/templates/conversation-view/conversation-view.html

```diff
@@ -9,7 +10,7 @@
 {% endif %}
 {% set ru_id = messages[0]['ru_id'] %}
 {% set survey = messages[0]['survey_id'] %}
-{% block page_title %} {{ subject }} | Survey Data Collection {% endblock %}
+{% set page_title = subject %}
```

Comment:
Doesn't this change the whole page title?

Reply:
Yes but `| Survey Data Collection` is appended in the base template as it's on every page.